src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -748,7 +763,7 @@ CHIP_ERROR Encode(chip::TLV::TLVWriter & writer, chip::TLV::Tag tag) const
     static void OnSuccessFn(void * context, id value) { DispatchSuccess(context, value); }
 };
 
-template <typename DecodableValueType> class BufferedReadClientCallback final : public app::ReadClient::Callback {
```

Comment:
Why this change?

Reply:
In order to add the above "BufferedReadClientCallback" property, the above class definition was additionally required. If final is not removed, a build error occurs.